dataclasses_avroschema/serialization.py

```diff
@@ -73,7 +109,7 @@ def to_json(data: typing.Dict[str, typing.Any]) -> typing.Dict:
             value = date_to_str(value)
         elif isinstance(value, datetime.time):
             value = time_to_str(value)
-        elif isinstance(value, uuid.UUID):
+        elif isinstance(value, uuid.UUID) or isinstance(value, decimal.Decimal):
```

Comment:
`isinstance` cac receive a tuple and you can avoid the `or`: `isinstance(value, (uuid.UUID. decimal.Decimal)`